platform/mellanox/fw.mk

```diff
@@ -1,6 +1,10 @@
 # mellanox firmware
 
-MLNX_FW_VERSION = 13.2000.1420
+# Stephen Sun comments out for sdk/sai integration
+# MLNX_FW_VERSION = 13.2000.1420
+#
+MLNX_FW_VERSION = 13.2000.2104
+
 MLNX_FW_FILE = fw-SPC-rel-$(subst .,_,$(MLNX_FW_VERSION))-EVB.mfa
```

Comment:
the url should be changed since it not upstreamed yet.

Reply:
fixed.